lib/index.js

```diff
@@ -16,15 +16,15 @@ var runQuery = function (credentials, method) {
     var p = new Promise(function(resolve, reject) {
       var success = function(results) {
         if (typeof cb === 'function') {
-          cb.apply(null, [null].concat(Array.prototype.slice.call(arguments)));
+          cb(null, results);
         }else{
```

Comment:
It think we should use a linter or something to well define the code style for this project. But yes, I think it's more sexy with space!